corehq/apps/export/tests/test_export_data_schema.py

```diff
@@ -539,9 +539,12 @@ def setUpClass(cls):
         cls.first_build.copy_of = cls.current_app.get_id
         cls.first_build.version = 3
 
+        cls.advanced_app = Application.wrap(cls.get_json('advanced_application'))
```

Comment:
don't think is necessary to change but if easy I find it more easy to reason with tests when the app is created by hand. E.g. using `AppFactory` or just manually adding stuff. E.g. something [like this](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/test_models.py#L23).